modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/url/UrlGenerator.java

```diff
@@ -50,6 +50,11 @@ private static String normalizePath( final String path )
             return "";
         }
 
+        if ( path.startsWith( "?" ) )
```

Comment:
why do we have a special branch for `?`. do we even support query string in path?

Reply:
Yes, for now, we are adding query params using a path strategy. Here, I used a special branch for `?` to avoid `/?`.